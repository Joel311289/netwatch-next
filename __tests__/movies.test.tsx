import { render } from '@testing-library/react';
import Movies from '@/pages/movies';

jest.mock('react-i18next', () => ({
  useTranslation: () => ({ t: (key: string) => key })
}));

describe('Tests movies page', () => {
  test('renders page correctly', () => {
    const { container } = render(<Movies />);

    expect(container).toMatchSnapshot();
  });
});
