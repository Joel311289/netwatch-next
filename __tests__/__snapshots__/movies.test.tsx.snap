// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests movies page renders page correctly 1`] = `
<div>
  <div
    class="containerWrapper hasMargins"
  >
    <h1>
      movies page
    </h1>
  </div>
</div>
`;
